libraries/standard/http/include/http_client.h

```diff
@@ -558,6 +629,8 @@ HTTPStatus_t HTTPClient_AddRangeHeader( HTTPRequestHeaders_t * pRequestHeaders,
  * @param[in] reqBodyBufLen The length of the request entity in bytes.
  * @param[in] pResponse The response message and some notable response
  * parameters will be returned here on success.
+ * @param[in] pFlags Flags noting other options to perform when sending the request
```

Comment:
The use of "other options" is ambiguous. 
(Nit recommendation): Flags to configure the behavior of the function when either sending the request or receiving the response. Refer to `#HTTP_SEND_DISABLE_CONTENT_LENGTH_FLAG` and `#HTTP_REQUEST_KEEP_ALIVE_FLAG` for supported flags.

Reply:
Updated.